### ISSUE TITLE
A title is currently emulating??

### ISSUE THREAD

**Issue by maxamillionrax** [Cxbx-Reloaded/Cxbx-Reloaded#1743]
okay so I have little to no coding knowledge so please be gentle with me, i thank you in advance. So i was playing jet set radio future last night on the emulation however i closed out the emulator by accident (i know im an idiot). So now when i try to run the  emulator it says "emulation already running blah blah close emulation before starting it up again" but nothing is running...i try to close the game and re open it but that does nothing. Can somebody PLEASE help me? I have loved this game since i could see with my eyeballs and hear with my ears and this is the first time i got to play it in about 16 years or so.

**Comment by LukeUsher** [comment]
The emulator probably didn't close properly, check task manager to make sure there isn't another instance of Cxbx.exe running.

And if there isn't, try restarting your computer.

**Comment by maxamillionrax** [comment]
> The emulator probably didn't close properly, check task manager to make sure there isn't another instance of Cxbx.exe running.
> 
> And if there isn't, try restarting your computer.

Thank you for that atleast fixed the issue of the emulator starting. Now my only issue is that the emulator starts and the screen is just black. How do i fix this now?

**Comment by ergo720** [comment]
That's a separate issue than the first one (although a common one and usually caused by corrupted emulator cache files and/or bad game dump). This a is also not a support forum to ask for help. For that, please use our discord channel [here](https://discordapp.com/invite/26Xjx23). Closing.

**Comment by AzurikRiseOfPerathia** [comment]
@maxamillionrax You know there is another way to say it!
